spacy/tests/regression/test_issue5314.py

```diff
@@ -0,0 +1,10 @@
+from bin.wiki_entity_linking.wikipedia_processor import _process_wp_text
+
+title = "Arkæologi"
+
+text = """<text bytes="11456" xml:space="preserve">[[Fil:Archäologie schichtengrabung.jpg|thumb|Arkæologisk [[udgravning]] med profil.]] '''Arkæologi''' er studiet af tidligere tiders [[menneske]]lige [[aktivitet]], primært gennem studiet af menneskets materielle levn.</text>"""
```

Comment:
Actually, to ensure this works also for the previous version, you could run the tests on a few different `text` versions, by using `parameterize` and providing a list of texts. Then we can explicitely test once with and once without the `bytes` parameter. See https://github.com/explosion/spaCy/blob/e7ba0c4e76d316e795d97d5a2729b83adeb51b43/spacy/tests/regression/test_issue1501-2000.py#L115 for an example of using `parameterize`.